imageresizerapp/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,7 @@
+﻿Install-ChocolateyPackage `
+    -PackageName 'imageresizerapp' `
+    -FileType 'EXE' `
+    -SilentArgs '/quiet /norestart' `
+    -Url 'https://github.com/bricelam/ImageResizer/releases/download/v3.0/ImageResizerSetup-3.0.exe' `
```

Comment:
The `Url64` parameter was originally intended for packages with separate 32- and 64-bit installers. There's a whole bitness detection routine and flags like `-forcex86`.

The `Url` parameter means "default" and "x86" and Chocolatey's message is a little misleading. It says something like

> Installing "foo" 32-bit

If your installer is actually multi-architecture or universal or whatever, you can get around it by passing the same values to all of the various Url parameters. Something like

```powershell
$url = 'http://foo.com/foo.msi
$checksum = '###################################'
$checksumType = 'SHA256'

Install-ChocolateyPackage `
  -PackageName 'foo' `
  -Url $url `
  -Url64 $url `
  -Checksum $checksum `
  -Checksum64 $checksum `
  -ChecksumType $checksumType `
  -ChecksumType64 $checksumType
```